moveit_setup_assistant/src/widgets/planning_groups_widget.cpp

```diff
@@ -282,6 +282,7 @@ void PlanningGroupsWidget::loadGroupsTree()
     btn_delete_->hide();
   }
 
+  current_edit_group_.clear();
```

Comment:
these kinds of changes should be in a separate PR... this one is way too big already

Reply:
I wasn't so sure too, but from the feedback I got from PR where I removed the unused variable, "it is advised not to open too small PR"

Reply:
[PR 924](https://github.com/ros-planning/moveit/pull/924)

Reply:
Yes there's a middle ground :-)

One line change is a bit small, but many hundreds of lines is getting large.